Physics.cpp

```diff
@@ -0,0 +1,23 @@
+#include <SFML/Graphics.hpp>
+#include <Box2D/Box2D.h>
+
+#include <iostream>
+
+#include "Physics.hpp"
+
+Physics::Physics( float x, float y, float width, float height, float density, float friction, b2BodyType bodyType, b2World * world ) {
```

Comment:
Tutaj pierdoła ale jak możesz to nie rób takich długich linii bo na mniejszym ekranie (lub podzielonym) ciężej to czytać.
Wstaw tutaj nową linię w którymś miejscu.